boards/arm/efm32pg_stk3402a/efm32pg_stk3402a.yaml

```diff
@@ -12,6 +12,7 @@ supported:
   - i2c
   - gpio
   - nvs
+  - watchdog
```

Comment:
Please also add the watchdog to the board documentation index.rst file, see https://github.com/chrta/zephyr/commit/f75080cda208e0cfd70966d43ef0c4eabdd0826b

Reply:
Added the watchdog entry to the documentation of the Pearl Gecko starter kit